src/main/java/com/redhat/devtools/intellij/lsp4mp4ij/psi/internal/graphql/MicroProfileGraphQLConstants.java

```diff
@@ -23,10 +23,12 @@ private MicroProfileGraphQLConstants() {
 
     public static final String QUERY_ANNOTATION = "org.eclipse.microprofile.graphql.Query";
     public static final String MUTATION_ANNOTATION = "org.eclipse.microprofile.graphql.Mutation";
-    public static final String SUBSCRIPTION_ANNOTATION = "org.eclipse.microprofile.graphql.Subscription";
```

Comment:
keep it and rename MICROPROFILE_SUBSCRIPTION_ANNOTATION

Reply:
Actually there is no `org.eclipse.microprofile.graphql.Subscription` class (see https://github.com/eclipse/microprofile-graphql/tree/main/server/api/src/main/java/org/eclipse/microprofile/graphql) - that was an error, thus changing the class name isn't really a breaking change.

Reply:
@datho7561 do you remember why we use this class `org.eclipse.microprofile.graphql.Subscription`  in JDT?

Reply:
I think it was my mistake, adding it via https://github.com/redhat-developer/intellij-quarkus/commit/d7a9ec706bfd90409f1b8be4df477543bf1186cd#diff-0800a063c095ddbc177c8d186ae4dcbe5299b3b65d6507f0541bbe6be2aa3084R26